packages/web/src/store/selectors/scanDisplayOptions/packageFilters.ts

```diff
@@ -0,0 +1,22 @@
+import { PackageTrait } from '../../slices/scanDisplayOptions';
+import { IdentifiedPackage } from '../websiteResults';
+
+export type OptimizedPackageFilters = {
```

Comment:
Are there non-optimized filters somewhere?

Reply:
In the store itself, in raw serializable form.